src/ext/resources/legacygraphqlapi/schema.graphqls

```diff
@@ -1340,6 +1340,36 @@ type Itinerary {
     systemNotices: [SystemNotice]!
 }
 
+"An amount of money."
+type Money {
+    "ISO-4217 currency code"
+    currency: String!
+    """
+    Money amount in cents. **Note:** this value is dependent on the currency used,
+    as one cent is not necessarily ¹/₁₀₀ of the basic monetary unit.
+    """
+    cents: Int!
+}
+
+""""
+An ISO-8601-formatted duration, i.e. `PT2H30M` for 2 hours and 30 minutes.
+"""
+scalar Duration
+
+"An estimate for a ride on a hailed vehicle, like an Uber car."
+type RideHailingEstimate {
+    "The provider of the ride hailing service."
+    provider: String!
+    "The lower bound of the price estimate of this ride."
+    minPrice: Money!
+    "The upper bound of the price estimate of this ride."
+    maxPrice: Money!
+    "The estimated time it takes for the vehicle to arrive."
+    arrival: Duration!
+    "The name of the ride, ie. UberX"
+    name: String
```

Comment:
Having a name for an estimate is a bit strange...

Reply:
I know, I just tried to be funny ;-) It break basic naming conventions - it should be called "productName" or something like that?

Reply:
Done in f07d1536d05e842f2abd1a21beeddf1f212eec68